app/controllers/api/v1/mixins/index_mixin.rb

```diff
@@ -36,7 +36,11 @@ def access_relation(relation)
             relation.by_owner
           else
             ids = ace_ids('read', relation.model)
-            ids.any? ? relation.where(:id => ids) : relation
+            if relation.model.respond_to?(:aceable?) && relation.model.aceable?
```

Comment:
Can this PR include a screen shot showing the difference between an `org_admin` logging in with no previous shares and a normal user logging in with no previous shares. That will more easily show what this change accomplishes.

Reply:
@syncrou As discussed at the standup, there is no different between org_admin and a normal user when it comes to sharing.  The user would need to be in a group with the catalog_admin role to be able to see all Portfolios if they have not been shared with the user.

Minor, but could this be `if relation.model.try(:aceable?)`